ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/templates/scheming/organization/group_form.html

```diff
@@ -23,8 +23,29 @@
     {%- endfor -%}
     {{ form.hidden('csrf-token', h.generate_csrf_token()) }}
 
+    <div class="form-actions">
+        {% block delete_button %}
+        {% if action == 'edit' %}
+          {% if h.check_access('group_delete', {'id': data.id}) and action=='edit'  %}
```

Comment:
This should be `h.check_access('organization_delete'`

Reply:
Updated